charts/longhorn/102.2.1+up1.4.1/Chart.yaml

```diff
@@ -0,0 +1,40 @@
+annotations:
+  catalog.cattle.io/auto-install: longhorn-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>= 1.21.0-0'
+  catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
```

Comment:
Not sure if we should change this one to longhorn.io/v1beta2 since we have been already serving that version for a long time. Maybe we can create a ticket to investigate?

Reply:
Where are these annotations from?

Reply:
These annotations originate from past changes and are mentioned in the current Longhorn documentation, but I think this time they should be corrected to `longhorn.io/v1beta2` to respect the Longhorn 1.4.1 API version. If this annotation is updated, I would check that the system is functional.

@PennyScissors Would you mind to give us any further document to mention these annotations' purpose or the usage to connect Rancher and Longhorn?